frontend/src/index.ts

```diff
@@ -157,6 +153,10 @@ export class App extends BtrixElement {
   willUpdate(changedProperties: Map<string, unknown>) {
     if (changedProperties.has("settings")) {
       AppStateService.updateSettings(this.settings || null);
+
+      if (this.settings && !changedProperties.get("settings")) {
+        void localize.initLanguage();
```

Comment:
oh, should this block as well?

Reply:
I don't think we need to since the `@localized()` directive triggers a re-render that's imperceptible, unless you're seeing a flash of English?